ledger/acctupdates_test.go

```diff
@@ -3004,3 +2887,141 @@ func TestAcctUpdatesLookupResources(t *testing.T) {
 	require.Contains(t, data.Assets, aidx3)
 	require.NotContains(t, data.Assets, aidx2)
 }
+
+func TestAcctUpdatesLookupStateDelta(t *testing.T) {
```

Comment:
Could you add a comment that describes this test and/or some more detailed inline comments about how the test works?

Reply:
Added a bunch of comments--hopefully that makes it a bit easier to parse.